src/DefaultConnectionPainter.cpp

```diff
@@ -251,4 +199,55 @@ void ConnectionPainter::paint(QPainter *painter, ConnectionGraphicsObject const
     painter->drawEllipse(cgo.in(), pointRadius, pointRadius);
 }
 
+QPainterPath DefaultConnectionPainter::getPainterStroke(ConnectionGraphicsObject const &connection) const
+{
+    auto cubic = cubicPath(connection);
+
+    QPointF const &out = connection.endPoint(PortType::Out);
+    QPainterPath result(out);
+
+    unsigned segments = 20;
```

Comment:
This could be `constexpr`, unless you plan for this to be variable?

Reply:
Fixed